azure-toolkit-libs/azure-toolkit-containerapps-lib/src/main/java/com/microsoft/azure/toolkit/lib/containerapps/containerapp/ContainerAppDraft.java

```diff
@@ -129,12 +130,22 @@ public com.azure.resourcemanager.appcontainers.models.ContainerApp createResourc
             .withTemplate(template)
             .withWorkloadProfileName(workloadProfile)
             .create();
-        final Action<ContainerApp> updateImage = AzureActionManager.getInstance().getAction(ContainerApp.UPDATE_IMAGE).bind(this);
-        final Action<ContainerApp> browse = AzureActionManager.getInstance().getAction(ContainerApp.BROWSE).bind(this);
+        final Action<ContainerApp> updateImage = Optional.ofNullable(AzureActionManager.getInstance().getAction(ContainerApp.UPDATE_IMAGE))
+            .map(action -> action.bind(this))
+            .orElse(null);
```

Comment:
Instead of returning null, should we just return a no-op action? Otherwise we need to handle nulls in a lot of other places.

We can predefine a constant and check for identity equal if necessary.